### DISCUSSION TITLE
[BUG] Missing pnp trace log file - Set-PnPTraceLog -On -Level Debug -LogFile "somePath/pnpTemplateProvisioningLogging-someDateTimeStamp.txt"

### DISCUSSION THREAD

**Discussion by hajjkh** [pnp/powershell#3901] [General]
I do some PnP PowerShell logic like SharePoint site provisioning. I would like to save the pnp debug log in a file on a specific path. I try the code below, but I dont see a log file. 

`Set-PnPTraceLog -On -Level Debug -LogFile "C:\PS\Diversen\DEV\azFunc\logs\pnpTemplateProvisioning\pnpTemplateProvisioningLogging-someDateTimeStamp.txt"`

### Expected behavior 
I expect a logging file on a specific path

### Actual behavior 
I see nothing

### Steps to reproduce behavior
```
Set-PnPTraceLog -On -Level Debug -LogFile "C:\PS\Diversen\DEV\azFunc\logs\pnpTemplateProvisioning\pnpTemplateProvisioningLogging-someDateTimeStamp.txt"
Invoke-PnPSiteTemplate -Path $templatePath
```

### What is the version of the Cmdlet module you are running?
2.4.0

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows

**Comment by gautamdsheth** [comment]
Can't reproduce this issue, moving to discussion.
